### PR TITLE
EDU-989: add update item to Go feature TOC

### PR DESCRIPTION
**Pull request by DJSanti**
## What does this PR do?
This PR provides a link to the Updates section of the Go Dev Guide

### CODE REVIEW

File: docs-src/go/features.md [hunk 1]
```diff
@@ -25,3 +25,4 @@ In this section you can find the following:
 - [How to Asynchronously complete an Activity](#asynchronous-activity-completion)
 - [How to register Namespaces](#namespaces)
 - [How to use custom payload conversion](#custom-payload-conversion)
+- [How to develop updates](#updates)
```
Comment (flossypurse) [inline docs-src/go/features.md:28]: This is a source node - so you can use the id of the node, and link magic rewrites -

File: package.json [hunk 2]
```diff
@@ -47,6 +47,7 @@
     "clsx": "^1.1.1",
     "fs-extra": "^10.1.0",
     "gray-matter": "^4.0.3",
+    "latest": "^0.2.0",
```
Comment (flossypurse) [inline package.json:50]: what is this?

**Review by flossypurse — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (DJSanti) [reply package.json:50]: not sure

**Review by flossypurse — CHANGES_REQUESTED** [review]
The are 2 places where the link needs to be added.

File: docs-src/go/features.md [hunk 3]
```diff
@@ -25,3 +25,4 @@ In this section you can find the following:
 - [How to Asynchronously complete an Activity](#asynchronous-activity-completion)
 - [How to register Namespaces](#namespaces)
 - [How to use custom payload conversion](#custom-payload-conversion)
+- [How to develop updates](/go/updates)
```
Comment (djmagee) [inline docs-src/go/features.md:28]:
```suggestion
- [How to develop with Updates](/go/updates)
```

File: yarn.lock [hunk 4]
```diff
@@ -3743,7 +3743,7 @@ combine-promises@^1.1.0:
 
 combined-stream@^1.0.8:
   version "1.0.8"
-  resolved "https://registry.yarnpkg.com/combined-stream/-/combined-stream-1.0.8.tgz#c3d45a8b34fd730631a110a8a2520682b31d5a7f"
```
Comment (flossypurse) [inline yarn.lock:3746]: We use yarn.
So, I don't know why this is changing to npmjs ?

Need to back this out.

**Review by flossypurse — CHANGES_REQUESTED** [review]

**Pull request merged**